packages/backend/src/modules/tvl/repositories/deleteArchivedRecords.ts

```diff
@@ -7,17 +7,20 @@ import { Knex } from 'knex'
 export function deleteHourlyUntil(
   knex: Knex,
   tableName: string,
-  timestamp: UnixTime,
+  to: UnixTime,
+  from: UnixTime | undefined,
 ) {
-  return (
-    knex(tableName)
-      .where('unix_timestamp', '<', timestamp.toDate())
-      // do not delete daily
-      .andWhereRaw(`extract(hour from "unix_timestamp") != 0`)
-      // do not delete six hourly
-      .andWhereRaw(`extract(hour from "unix_timestamp") % 6 != 0`)
-      .delete()
-  )
+  let query = knex(tableName)
+    .where('unix_timestamp', '<', to.toDate())
+    // do not delete six hourly and daily
+    .andWhereRaw(`extract(hour from "unix_timestamp") % 6 != 0`)
+    .delete()
+
+  if (from) {
+    query = query.andWhere('unix_timestamp', '>=', from.toDate())
```

Comment:
Just double-checking - are you certain it should be `>=` and not `>` ?

Reply:
We are deleting till exclusively so since needs to include the record.